### PR TITLE
Added fs-fuzz test

### PR DESCRIPTION
**Pull request by PraveenPenguin**
fs-fuzz : two simple fuzzers
one for file system , second one for c stream

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: fs/fs-fuzz.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+        # Check for basic utilities
+        smm = SoftwareManager()
+
+        for package in ['make', 'gcc']:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.cancel('%s is needed for the test to be run' % package)
+
+        tarball = self.fetch_asset(
+            'https://github.com/regehr/fs-fuzz/archive/master.zip')
```
Comment (narasimhan-v) [inline fs/fs-fuzz.py]: Please do not use generic name master.zip.
Reason: https://github.com/avocado-framework-tests/avocado-misc-tests/pull/953/commits/5256d345d00cfcccee6c430992c906eb434f5719

File: fs/fs-fuzz.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        self.build_dir = os.path.join(self.workdir, 'fs-fuzz-master')
+        build.make(self.build_dir)
+        self.clear_dmesg()
+
+    def test(self):
+        os.chdir(self.build_dir)
+
+        if process.system('./fd_fuzz 1', shell=True, ignore_status=True) != 0:
+            self.fail("stress test for  file I/O layer failed")
+        if process.system('./file_fuzz 1', shell=True, ignore_status=True) != 0:
```
Comment (narasimhan-v) [inline fs/fs-fuzz.py]: I would suggest adding 2 separate `test_x()` for this.

Comment (PraveenPenguin) [reply fs/fs-fuzz.py]: @narasimhan-v ah yes cool suggestion will adopt

**Comment by PraveenPenguin** [comment]
@narasimhan-v adopted the change please have a look

**Comment by PraveenPenguin** [comment]
job log:

```[root@f200-lp4 ~]# avocado run fs-fuzz.py 
Failed to load plugin from module "avocado_vt.plugins.vt_list": EnvironmentError("Bootstrap missing. Execute 'avocado vt-bootstrap' or disable this plugin to get rid of this message",)
Failed to load plugin from module "avocado_vt.plugins.vt": EnvironmentError("Bootstrap missing. Execute 'avocado vt-bootstrap' or disable this plugin to get rid of this message",)
JOB ID     : 94083c85bf34ffafacde9382820629f88a5838ea
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2018-07-06T02.30-94083c8/job.log
 (1/2) fs-fuzz.py:FsFuzz.test_fd: PASS (1734.53 s)
 (2/2) fs-fuzz.py:FsFuzz.test_file: PASS (1496.14 s)```

**Pull request merged**